src/components/account/README.md

```diff
@@ -0,0 +1 @@
+An account system is responsible for user accounts and having a location for where the data root is stored. This allows for the file system to be loaded on another device, even if the related devices are offline.
```

Comment:
I moved the explanations of each component to a README file in their respective directory, since we no longer have those component variables in the main index file.